wg-naming/README.md

```diff
@@ -0,0 +1,70 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Naming Working Group
+
+
+## Stakeholder SIGs
+* SIG Architecture
+* SIG Contributor Experience
+* SIG Docs
+
+
+
+## Organizers
+
+* Celeste Horgan (**[@celestehorgan](https://github.com/celestehorgan)**), CNCF
+* Stephen Augustus (**[@justaugustus](https://github.com/justaugustus)**), VMware
+* Zach Corleissen (**[@zacharysarah](https://github.com/zacharysarah)**), Linux Foundation
+
+## Contact
+- Slack: [#wg-naming](https://kubernetes.slack.com/messages/wg-naming)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-naming)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/wg%2Fnaming)
+<!-- BEGIN CUSTOM CONTENT -->
+
+**The following section will be reworked and formalized as a charter once the
+Working Group has been approved by the Steering Committee.**
+
+### Goals
+
+- Evaluate language and naming choices within the Kubernetes organization, with
+  a specific initial focus of:
+  - Removing language which cannot be fully separated from racist, sexist or
+    homophobic connotations in cultural contexts outside of computer
+    programming
+  - Improving clarity of codebases and documentation by replacing idioms,
+    metaphors and slang specific to the English language
+- Create a list of terms to avoid and propose alternatives
+- Define a process by which any member of the Kubernetes organization can
+  propose a language recommendation, have them evaluated based on a rubric, and
+  see them implemented across all codebases.
+  - Provide an easily findable location for language recommendations and
+    follow-up issues, similar to an architectural decision record.
+- Work with stakeholder SIGs to implement the changes recommended. We
+  anticipate the following:
+  - Provide stakeholder SIGs with guidance on naming, language conventions, and
+    processes
+  - Collaborate with SIG Architecture and other stakeholders on an
+    implementation timeline and strategies for dealing with follow-up issues
+    from renaming, like deprecations
+  - Collaborate with SIG Docs and SIG Contributor Experience on documenting
+    language recommendations and processes
```

Comment:
Would this also involve renaming the master branch for all repos and would renaming the branch be a criteria for dissolution? It's heavily tied to how quickly and widely GitHub responds so I just want us to be cognizant of that.

Reply:
At the moment, CNCF is planning to wait for GitHub to roll out changes regarding master branch naming. We're hoping that waiting on GitHub to make changes centrally results in less tooling issues down the line.

Ultimately the choice lies with the community, but I would strongly recommend Kubernetes take the same tactic. Personally I would not include this discussion in the WG's domain – the WG should focus on Kubernetes and related projects only, not the wording in the tooling, code language or libraries Kubernetes uses.

Reply:
Yes, we're waiting for GitHub's lead here, and they should have a response soon.
(Discussed during the 6/25 [GitHub Administration meeting](https://docs.google.com/document/d/1IiVrr1hcFWmbboExk971FsMUGfr2Wp68mdMribCuzLs/edit#heading=h.5wzcacxwipf3).)

I'm taking point on this from the WG Naming side and will open an issue for this later in the week.

Reply:
(Added as a post-formation discussion point.)